doc/userguide/rules/index.rst

```diff
@@ -26,6 +26,7 @@ Suricata Rules
    snmp-keywords
    base64-keywords
    sip-keywords
+   smb-keywords
```

Comment:
Thanks for adding the doc.

If you are interested, there is also https://redmine.openinfosecfoundation.org/issues/3030 ;-)